docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -198,8 +213,154 @@ List<String> namesButEmmanuels = persons
 
 NOTE: The `stream` methods require a transaction to work.
 
+=== Adding entity methods
+
+In general, we recommend not adding custom queries for your entities outside of the entities themselves,
```

Comment:
style wise, try and keep your sentences light. By definition what you write is what you recommend.
Also "in general", "usually" etc bring no value.
Also be positive, negations are always triggering some negative loops in the brain
Keep sentences short.

Add custom queries on your entities outside of the entities themselves. 
That way, you and your co-workers can find the easily and queries are right next to the object they operate on.
Adding them as static methods in your entity class is the Panache Active Record way.